### PR TITLE
Allow user to specify tf.keras style loss function

### PR DESCRIPTION
**Pull request by terrytangyuan**
This addresses the third issue on `loss` in https://github.com/sql-machine-learning/elasticdl/issues/1476 to support loss functions from `tf.keras.losses`. Also added a test case.

cc: @typhoonzero 

Signed-off-by: terrytangyuan <terrytangyuan@gmail.com>

### CODE REVIEW

**Review by typhoonzero — APPROVED** [review]
`keras_loss(labels, predictions)` is compatible with keras model.compile now
